src/resources/js/components/Presentation.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="container">
```

Comment:
please follow the styleguide this becomes `<v-container>`

Reply:
ok, I've added styleguide ``<v-contanier>`` markup in [3b87ebbf](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/113/commits/3b87ebbffe5bb3916f953f2febccd8bed753c3e3)